doc/schemas/listconfigs.schema.json

```diff
@@ -33,6 +33,16 @@
             "required": [],
             "description": "Specific options set for this plugin",
             "properties": {}
+          },
+          "internal_config": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "additionalProperties": true,
+              "required": [],
+              "description": "Specific internal option of this plugin",
+              "properties": {}
+            }
```

Comment:
Wouldn't it be better to just allow this to be a dict? Then plugins can just have key-value pairs instead of having to nest them into an array, which I think is the most common thing plugins will do. Alternatively we could allow `additionalProperties` on wrapping object, though that'd mean losing some checks.

Reply:
Dropped this part, probably for another PR. Yes dict would be nice.